src/containers/Authorization/Navigation.tsx

```diff
@@ -4,7 +4,8 @@ import { useTranslation } from 'react-i18next';
 import { styled } from '../../styles';
 import { ButtonText } from '../../components';
 
-import { TabConst } from './Authorization';
+import { TabConst } from '.';
```

Comment:
mmm... i would suggest, to move const to the file with types. because enum is kind of type) so this is valid case